charts/datadog/values.yaml

```
@@ -557,6 +559,9 @@ clusterAgent:
   # clusterAgent.enabled -- Set this to false to disable Datadog Cluster Agent
   enabled: true
 
+  # clusterAgent.shareProcessNamespace -- Set the process namespace sharing on the Datadog Cluster Agent
+  shareProcessNamespace: true
```

same comment about default value to false 

```suggestion
  shareProcessNamespace: false
```